parquet-thrift/src/main/java/org/apache/parquet/thrift/ThriftRecordConverter.java

```diff
@@ -445,7 +447,16 @@ public FieldEnumConverter(List<TProtocol> events, ThriftField field) {
 
     @Override
     public void addBinary(final Binary value) {
-      final int id = enumLookup.get(value);
+      final Integer id = enumLookup.get(value);
+
+      if (id == null) {
```

Comment:
FYI, as far as I can tell, this implementation is faster than:

```
try {
  final int id = enumLookup.get(value);
} catch (NullPointerException e) {
  throw new ParquetDecodingException(...)
}
```

I thought maybe the above would be better because there's sort of a hint to the branch predictor that the 'normal' case is for this not to throw so predict that... but i did a small benchmark and the if statement seems to be faster